extensions/libraries/redcore/model/admin/legacy.php

```diff
@@ -9,96 +9,13 @@
 
 defined('JPATH_REDCORE') or die;
 
-if (version_compare(JVERSION, '3.0', 'lt'))
```

Comment:
Fields will populate `JTable` `updateAuditFields()`.
I don't see any reason why can be skip `updateAuditFields()` in joomla < 3.
Did you checked it in the real site or it is just your assumption?